SmartDeviceLink/SDLErrorConstants.h

```diff
@@ -199,3 +199,11 @@ typedef NS_ENUM(NSInteger, SDLCacheFileManagerError) {
      */
     SDLCacheManagerErrorUpdateIconArchiveFileFailure = -1,
 };
+
+typedef NS_ENUM(NSInteger, SDLAudioStreamManagerError) {
+    /// The audio stream is not currently connected
+    SDLAudioStreamManagerErrorNotConnected = -1,
+
+    /// Attempted to play but there's no audio in the queue
+    SDLAudioStreamManagerErrorNoQueuedAudio = -2
```

Comment:
`SDLAudioStreamManagerErrorNoQueuedAudio` does not seem to be used anywhere.

Reply:
I know, but it's a public API so removing it is a major version change.